tests/layer_tests/tensorflow_tests/test_tf_Size.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestSize(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'input' in inputs_info
+        input_shape = inputs_info['input']
+        input_type = self.input_type
+        inputs_data = {}
+        inputs_data['input'] = np.random.randint(-50, 50, input_shape).astype(input_type)
+
+        return inputs_data
+
+    def create_size_net(self, input_shape, input_type, out_type):
+        self.input_type = input_type
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            input = tf.compat.v1.placeholder(input_type, input_shape, 'input')
+            tf.raw_ops.Size(input=input, out_type=out_type)
```

Comment:
Could you check this place. What do we test in case of out_type==None? Because regarding https://www.tensorflow.org/api_docs/python/tf/raw_ops/Size default value is int32. I may expect we need to test some "default behavior" in case of None. In such case we need to add here a condition, like we do it previously.

Reply:
`None` means the default behavior or value is undefined for this parameter.
Yes, we did in the different way previously but I think it the same. TF accepts it.